app/helpers/blacklight/configuration_helper_behavior.rb

```diff
@@ -87,13 +87,11 @@ def document_show_field_label document, field
   # Look up the label for the facet field
   def facet_field_label field
     field_config = blacklight_config.facet_fields[field]
+    defaults = [:"blacklight.search.fields.facet.#{field}", :"blacklight.search.fields.#{field}"]
+    defaults << field_config.label if field_config
+    defaults << field.to_s.humanize
 
-    field_label(
-      :"blacklight.search.fields.facet.#{field}",
-      :"blacklight.search.fields.#{field}",
-      (field_config.label if field_config),
-      field.to_s.humanize
-    )
+    field_label defaults
   end
```

Comment:
This has changed the semantics when `field_config` is nil (or "falsy").  

In the first version, if `field_config` is falsy, there will be a nil third argument to `field_label`. 

In the new version, if `field_config` is falsy, there will only be three arguments passed to `field_label`, with `field.to_s.humanize` being the third rather than the fourth. 

Was that intended? Is it related to the 4.2.1 breakage somehow?